### ISSUE TITLE
Upgrade to 2.4.2 / 2.4.x

### ISSUE THREAD

**Issue by sfrooster** [dsherret/ts-simple-ast#54]
I tried doing this in my local fork and running the test so I could submit a PR, but the tests blow-up pretty bad.

I have other builds which I suspect are failing because of the 2.4.1 peer dependency.  Upgrading to 2.4.2 and/or changing the peer dependency to 2.4.x/2 ought to do it.

**Comment by dsherret** [comment]
Implemented in 0.48.2.

Most of the time I do get a few tests failing when upgrading the typescript version, but in this case it seemed to upgrade no problem: https://travis-ci.org/dsherret/ts-simple-ast/builds/257948008

It would be nice to change the dependency to an approximate version (like 2.4.x), but I've found they introduce changes in behaviour to the compiler api in even patch releases... so I need to keep the dependency as an exact version unfortunately. It's one of the downsides to this library.
